README.md

```diff
@@ -4,6 +4,8 @@
     <img src="rosalution_title.svg" alt="Rosalution Logo and Title" width="400"/>
 </p>
 
+Is it preventing the double trigger?
```

Comment:
```suggestion
```

Reply:
Thanks for catching this. I applied your suggestion and removed it